projects/timer_challenge/src/main.c

```diff
@@ -0,0 +1,42 @@
+#include "interrupt.h"   // interrupts are required for soft timers
+#include "log.h"         // for printing
+#include "soft_timer.h"  // for soft timers
+#include "wait.h"        // for wait function
+
+#define COUNTER_PERIOD 500
```

Comment:
normally we add the suffix `_MS` to denote the units of the value (milliseconds here)